.github/pull_request_template.md

```diff
@@ -6,12 +6,13 @@ Please write a few sentences describing the overall goals of the pull request's
 
 ## PR checklist
 
-Please tick as appropriate and edit the reasons (e.g.: "backport: not needed because this is a new feature")
+Please add/remove strikethrough ~~ as appropriate, and add any relevant link/s to the end of the line.
 
-- [ ] **changelog** provided, or not required
-- [ ] **3.6 backport** done, or not required
-- [ ] **2.28 backport** done, or not required
-- [ ] **tests** provided, or not required
+- [ ] **changelog** provided, ~not required because:~
+- [ ] **dev PR** provided #, ~not required because:~
+- [ ] **3.6 PR** provided #, ~not required because:~
```

Comment:
A common case will be that a backport is required, but not initially done. Normally we only start working on the backport after the development PR is approved. So maybe initially it should be

```suggestion
- [ ] **3.6 PR** TODO | not required because 
```

Reply:
I am againist that since it adds complexity both to the user facing instuctions, as well to the parsing scripts. When something is in TODO  state we can keep the current notation, that the checkbox is unchecked and unmodified.